fe/fe-core/src/main/java/com/starrocks/statistic/StatsConstants.java

```diff
@@ -24,7 +24,7 @@ public class StatsConstants {
     public static final int STATISTIC_TABLE_VERSION = 3;
     public static final int STATISTIC_BATCH_VERSION = 4;
     public static final int STATISTIC_EXTERNAL_VERSION = 5;
-
+    public static final int STATISTIC_EXTERNAL_QUERY_VERSION = 6;
```

Comment:
what's the diff with STATISTIC_EXTERNAL_VERSION?

Reply:
STATISTIC_EXTERNAL_VERSION is used to collected statistics to fe, and fe will insert it to be.
STATISTIC_EXTERNAL_QUERY_VERSION is used to query the column statistics.